Validation/GlobalDigis/src/GlobalDigisAnalyzer.cc

```diff
@@ -741,7 +741,7 @@ void GlobalDigisAnalyzer::fillECal(const edm::Event &iEvent, const edm::EventSet
   bool validDigiES = true;
   if (!EcalDigiES.isValid()) {
     LogDebug(MsgLoggerCat) << "Unable to find EcalDigiES in event!";
-    validDigiES = false;
+    [[clang::suppress]] validDigiES = false;
```

Comment:
clang analyzer complains as we force set ` validDigiES = false;` at few line later